networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc

```diff
@@ -6,26 +6,33 @@ include::_attributes/common-attributes.adoc[]
 // Unique attribute for the version of the release notes.
 
 toc::[]
+ 
+As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift software-defined networking (SDN) plugin. 
 
-As a cluster administrator, you can migrate to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin. The following options are available for migration:
+[WARNING]
+====
+If your cluster uses the OpenShift SDN network plugin, you cannot upgrade your cluster to {product-title} 4.17. You must initially migrate your cluster to the OVN-Kubernetes plugin and then upgrade your cluster to {product-title} 4.17.
+
+Additionally, if your cluster uses a third-party container network interface (CNI) plugin, you cannot migrate your cluster to the OVN-Kubernetes network plugin. For a complete list of these CNI plugins, see link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] (Red{nbsp}Hat Knowledgebase).
+====
+
+The following methods exist for migrating from the OpenShift SDN network plugin to the OVN-Kubernetes plugin:
 
 Limited live migration (preferred method)::
 This is an automated process that migrates your cluster from OpenShift SDN to OVN-Kubernetes without service interruption.
 
 Offline migration::
-This is a manual process that includes some downtime. This method is primarily used for self-managed {product-title} deployments, and should be used in the event that you cannot perform a limited live migration to the OVN-Kubernetes network plugin.
+This is a manual process that includes some downtime. This method is primarily used for self-managed {product-title} deployments, and consider using this method when you cannot perform a limited live migration to the OVN-Kubernetes network plugin.
 
 [WARNING]
-====
-It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
+==== 
+Do not automate the migration from OpenShift SDN to OVN-Kubernetes with a script or another tool such as Red{nbsp}Hat Ansible Automation Platform. This might cause outages or crash your {product-title} cluster.
```

Comment:
We use script for offline migration.  Make sure the statement of "Do not automate the migration from OpenShift SDN to OVN-Kubernetes with a script" is for limited live migration only.